sdk/helper/keysutil/policy.go

```diff
@@ -53,6 +53,7 @@ const (
 	KeyType_ECDSA_P256
 	KeyType_ED25519
 	KeyType_RSA2048
+	KeyType_RSA3072
```

Comment:
This needs to go at the end, otherwise you will change the meaning of stored key types that already exist since the constants will no longer match.

Reply:
Hi @jefferai , thanks for taking a look into my PR. I have just updated the PR and moved the line at the end of the list in both files where it occurs.